recipes/kadmin_init.rb

```diff
@@ -25,18 +25,18 @@
 log 'create-krb5-db' do
   message 'Creating Kerberos Database... this may take a while...'
   level :info
-  not_if "test -e #{node['krb5']['kdc_conf']['realms'][default_realm]['database_name']}"
+  not_if { ::File.exist?(node['krb5']['kdc_conf']['realms'][default_realm]['database_name']) }
```

Comment:
This causes tests to pass by accident. If you run the tests on a machine which includes these files locally, it will fail. Why did you move away from the shell-based `test -e`?

Reply:
in our testing with kitchen-ec2 it was always skipping these steps.  I just re-tested and it worked as expected with the `test -e`, reverting.